src/KM_Controls.pas

```diff
@@ -3154,7 +3154,7 @@ function TKMNumericEdit.KeyEventHandled(Key: Word; Shift: TShiftState): Boolean;
   end;
 
   //We want these keys to be ignored by chat, so game shortcuts still work
-  if Key in [VK_F1..VK_F12, VK_ESCAPE, VK_TAB] then Result := False;
+  if Key in [VK_F1..VK_F12, VK_ESCAPE, VK_RETURN, VK_TAB] then Result := False;

```

Comment:
After a quick read :
Can you still send a chat message by pressing the return key? I'm quite sure that a lot of players enjoy that feature.
Should be tested to be sure. :-)